<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EndLineMessage</class>
 <widget class="QDialog" name="EndLineMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="message1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>已到达当前调查行的末尾。</string>
   </property>
  </widget>
  <widget class="QLabel" name="message2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>当前位置:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lineLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>LINE:</string>
   </property>
  </widget>
  <widget class="QLabel" name="markLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>MARK:</string>
   </property>
  </widget>
  <widget class="QLabel" name="message3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>按任意键清除本消息。</string>
   </property>
  </widget>
  <widget class="QLabel" name="message4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>之后按&quot;MARK&quot;键开始下一行采集</string>
   </property>
  </widget>
  <widget class="QLabel" name="message5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>或</string>
   </property>
  </widget>
  <widget class="QLabel" name="message6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>按&quot;MENU&quot;键并选择&quot;编辑LINE与MARK&quot;</string>
   </property>
  </widget>
  <widget class="QLabel" name="message7">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>进行数据编辑</string>
   </property>
  </widget>
  <widget class="QLabel" name="message8">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lineDataLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="markDataLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
